sdk/PrebidServerAdapter/PrebidServerModule/PBServerRequestBuilder.m

```diff
@@ -260,6 +261,17 @@ - (NSDictionary *)openrtbGeo {
     }
 }
 
+-(NSDictionary *) openrtbRegs {
+    
+    NSMutableDictionary *regsDict = [[NSMutableDictionary alloc] init];
+    
+    BOOL gdpr = [[PBTargetingParams sharedInstance] gdpr];
+    
+    regsDict[@"ext"] = @{@"gdpr" : @(@(gdpr).integerValue)};
```

Comment:
This should be do not send GDPR related info if Publisher does not set on the API.

Reply:
By default the consent is treated as Yes if the user has not provided the value so technically we can to pass in YES here